cli/server_request_command.go

```diff
@@ -113,6 +121,11 @@ func configureServerRequestCommand(srv *fisk.CmdClause) {
 	healthz := req.Command("jetstream-health", "Request JetStream health status").Alias("healthz").Action(c.healthz)
 	healthz.Flag("js-enabled", "Checks that JetStream should be enabled on all servers").Short('J').BoolVar(&c.jsEnabled)
 	healthz.Flag("server-only", "Restricts the health check to the JetStream server only, do not check streams and consumers").Short('S').BoolVar(&c.jsServerOnly)
+
+	profilez := req.Command("profile", "Run a profile").Action(c.profilez)
+	profilez.Arg("profile", "Specify the name of the profile to run (allocs, heap, goroutine, mutex, threadcreate, block)").StringVar(&c.profileName)
+	profilez.Arg("dir", "Set the output directory for profile files").Default(".").ExistingDirVar(&c.profileDir)
+	profilez.Flag("debug", "Set the debug level of the profile").IntVar(&c.profileDebug)
```

Comment:
this one *might* be problematic as we need to be quite careful with duplicate flags and this is very common - but lets leave as is and i will maybe update this before release